src/core/Object3D.js

```diff
@@ -762,6 +763,21 @@ Object3D.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 
 		}
 
+		if ( this.isSkinnedMesh ) {
+
+			if ( this.bindMode !== undefined ) object.bindMode = this.bindMode;
+			if ( this.bindMatrix !== undefined ) object.bindMatrix = this.bindMatrix.toArray();
```

Comment:
I don't think `.bindMode` and `.bindMatrix` can be `undefined` here. Yeah more robust tho.